api-reference/subscribers/get-subscribers.mdx

```diff
@@ -1,5 +1,155 @@
 ---
-openapi: get /v1/subscribers
+openapi: get /v1/subscribers/{subscriberId}
```

Comment:
```suggestion
openapi: get /v1/subscribers
```
I think this will not need the `subscriberId`
@sumitsaurabh927 can you confirm this?

Reply:
Yes, it it not needed!